src/Breadcrumb/index.jsx

```diff
@@ -3,29 +3,30 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 import Icon from '../Icon';
+import { ChevronRight } from '../../icons';
 
 const Breadcrumbs = ({
   links, activeLabel, spacer, clickHandler,
 }) => {
   const linkCount = links.length;
 
   return (
-    <nav aria-label="breadcrumb">
-      <ol className={classNames('list-inline')}>
+    <nav aria-label="breadcrumb" className="breadcrumb-container">
```

Comment:
Please use the name `pgn__*` for paragon component class names. So in this case `pgn__breadcrumb-container`